example/transactions/FeeDel_11_TxTypeFeeDelegatedValueTransferMemo.js

```diff
@@ -0,0 +1,65 @@
+const ethers = require("ethers");
+const { KlaytnWallet } = require("../../dist/src/ethers"); // require("@klaytn/sdk-ethers");
+const { objectFromRLP } = require("../../dist/src/core/klaytn_tx");
+
+const fs = require('fs');
+const sender_priv = fs.readFileSync('./example/privateKey', 'utf8') // private key of sender 
+const feePayer_priv = fs.readFileSync('./example/feePayerPrivateKey', 'utf8') // private key of feeDelegator
+
+const sender = '0x3208ca99480f82bfe240ca6bc06110cd12bb6366' 
+const reciever = '0xc40b6909eb7085590e1c26cb3becc25368e249e9' 
+const feePayer = '0x24e8efd18d65bcb6b3ba15a4698c0b0d69d13ff7'
+
+const provider = new ethers.providers.JsonRpcProvider('https://public-en-baobab.klaytn.net')
+
+//
+// TxTypeFeeDelegatedValueTransferMemo
+// https://docs.klaytn.foundation/content/klaytn/design/transactions/fee-delegation#txtypefeedelegatedvaluetransfermemo
+// 
+//   type: Must be 0x11,
+//   nonce: In signTransactionAsFeePayer, must not be omitted, because feePayer's nonce is filled when populating
+// 
+
+async function doSender() {
+  const sender_wallet = new KlaytnWallet(sender_priv, provider);
+  
+  let tx = {
+      type: 0x11,         
+      gasLimit: 1000000000, 
```

Comment:
Please use a realistic number. See Klaytnscope or receipt for actual gas consumed, then add some margin to it.

Reply:
Good idea. 

In the TxTypeFeeDelegatedValueTransferMemo example, even if gasLimit is omitted, it is executable, so I deleted gasLimit field.

But the same issue exists in the TxTypeFeeDelegatedAccountUpdateWithRatio example, so I modified it to be realistic number which the gas is actually used.

https://github.com/blukat29/klaytn-sdk-design/pull/2/commits/1265baf27a256123d4a5bae5fcbdcf9495d8977c